overthebox/defaults/network.defaults

```diff
@@ -10,6 +10,14 @@ if [ $? -ne 0 ]; then
 EOF
 fi
 
+uci get network.globals.tcp_congestion_control >/dev/null 2>&1
```

Comment:
So we force the user to use only "cubic" ?
My main idea was to set "cubic" by default to override unexpected kernel default value like olia.